embedded-google-pubsub/src/main/java/com/playtika/test/pubsub/EmbeddedPubsubBootstrapConfiguration.java

```diff
@@ -62,18 +61,17 @@ public GenericContainer pubsub(ConfigurableEnvironment environment,
         log.info("Starting Google Cloud Pubsub emulator. Docker image: {}", properties.getDockerImage());
 
         GenericContainer container = new GenericContainer(properties.getDockerImage())
-                .withExposedPorts(properties.getPort())
-                .withCommand(
-                        "/bin/sh",
-                        "-c",
-                        format(
-                                "gcloud beta emulators pubsub start --project %s --host-port=%s:%d",
-                                properties.getProjectId(),
-                                properties.getHost(),
-                                properties.getPort()
-                        )
+            .withExposedPorts(properties.getPort())
+            .withCommand(
+                "/bin/sh",
+                "-c",
+                format(
+                    "gcloud beta emulators pubsub start --project %s --host-port=%s:%d",
+                    properties.getProjectId(),
+                    properties.getHost(),
+                    properties.getPort()
                 )
-                .waitingFor(new LogMessageWaitStrategy().withRegEx("(?s).*started.*$"));
+            );
```

Comment:
There should be some waiting strategy for the container to start. What is the replacement?

Reply:
`HostPortWaitStrategy` ist the default, waiting for the log wasn't necessary for me.

Reply:
@reneleonhardt what do you mean _for me_? @Aloren @tdanylchuk  this change broke my pipeline with the latest version upgrade, I really question you why there is such irrelevant question in the PR with dependencies upgrade

I open an issue and a PR to revert it back.